cannot assign a value to variable PyPlot.clf from module FD_SANDBOX_15301797944822562081
